src/State/ViewRequests/selectors.js

```diff
@@ -0,0 +1,2 @@
+// export const getInboundRequests = state => state.viewRequests.inboundRequests;
```

Comment:
I think you addressed it in the MR at a top level, but can you clarify why these are commented right now? Thanks